libs/components/indicators/testing/src/label/fixtures/label-harness-test.component.html

```diff
@@ -0,0 +1,17 @@
+<sky-label
+  id="label-with-label-type"
+  [labelType]="labelType"
+  [customDescription]="customDescription"
+  [descriptionType]="descriptionType"
+  [attr.data-sky-id]="'label-with-label-type'"
```

Comment:
Does this need to use Angular binding? Can you use `data-sky-id="label-with-label-type"` instead?